server/storage/schema/schema.go

```diff
@@ -131,8 +134,16 @@ var (
 		version.V3_6: {
 			addNewField(Meta, MetaStorageVersionName, emptyStorageVersion),
 		},
+		version.V3_5: {
+			// it's safe to set emptyValue
+			// UnsafeReadConsistentIndex, UnsafeConfStateFromBackend, DowngradeInfoFromBackend checks for zero-length values
+			addNewField(Meta, MetaTermKeyName, emptyValue),
+			addNewField(Meta, MetaConfStateName, emptyValue),
+			addNewField(Cluster, ClusterDowngradeKeyName, emptyValue),
+		},
 	}
 	// emptyStorageVersion is used for v3.6 Step for the first time, in all other version StoragetVersion should be set by migrator.
 	// Adding a addNewField for StorageVersion we can reuse logic to remove it when downgrading to v3.5
 	emptyStorageVersion = []byte("")
+	emptyValue          = []byte("")
```

Comment:
Think that `emptyValue` for v3.4 fields should behave the same way as `emptyStorageVersion` for v3.5